Jenkinsfile

```diff
@@ -52,7 +56,7 @@ pipeline {
             when { not { branch 'master' } }
             steps {
                 sh """
-                cmdstan_hash=\$(git submodule status | grep cmdstan | awk '{print \$1}')
+                cmdstan_hash=\$(if [ ${cmdstan_hash} -eq '' ]; then echo ${cmdstan_hash}; else git submodule status | grep cmdstan | awk '{print \$1}' fi)
```

Comment:
isn't this the opposite? don't you want -neq?

Reply:
It should be fine now.